FuelCalculator_programmatically/Model/TtripEntry.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  tripEntry.swift
+//  FuelCalculator_programmatically
+//
+//  Created by Roman Cebula on 08/03/2020.
+//  Copyright © 2020 Roman Cebula. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct TripEntry {
+  let tripName: String
+  let tripDistance: String
+  let fuelConsumption: String
+  let tripCost: String
```

Comment:
Whenever possible, you dedicated types.

```suggestion
  let tripName: String
  let tripDistance: CLLocationDistance // or just Double
  let fuelConsumption: Double // or create typealias FuelConsumption = Double
  let tripCost: Decimal // or https://github.com/danthorpe/Money
```